metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -182,11 +194,41 @@ def convert_to_aspects(
                 ]
             )
             aspect_map[Constants.ADD_OWNER_OPERATION] = owner_aspect
+
         if Constants.ADD_TERM_OPERATION in operation_map:
             term_aspect = mce_builder.make_glossary_terms_aspect_from_urn_list(
                 sorted(operation_map[Constants.ADD_TERM_OPERATION])
             )
             aspect_map[Constants.ADD_TERM_OPERATION] = term_aspect
+
+        if Constants.ADD_DOC_LINK_OPERATION in operation_map:
+            try:
+                docs_dic = ast.literal_eval(
```

Comment:
ast eval feels strange here - could we just use `json.loads` or similar instead?

also, the ownership one supports ownership types without ever doing any parsing - can we use the same approach of a nested object here?

Reply:
good call, went with the second option. Parsing was indeed unnecessary